### PR TITLE
feat: added `Connection troubles` modal

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: added connection troubles modal

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [x] tested on mobile devices

**Release notes:**

* feat: added connection troubles modal

**This pull request makes the following changes:**

**Adds**

* feat: added connection troubles modal

image

image

image

image

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Comment by Kahonnohak** [comment]
The button on the sidebar might be too generous to give the blue, please change;

default: #5f656f
hover:  the blue

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -417,6 +421,24 @@ export default {
...
+        connectionTroubles: 'Connection Troubles',
+        tipsDescription:
+          'There are many reasons why you are unable to connect. Try below if it did not get sorted ask member of the community.',
+        tryOtherEndpoints: 'Try other endpoints',
+        changeFromHeader: 'Change the endpoint from network button on the header',
+        clearLocalStorage: 'Clear the Local Storage',
+        clearLocalStorageTip: 'This means your cache of the Portal on this browser will be cleared',
+        metaUpdate: 'Metadata Update on Polkadot.js app',
+        metaUpdateTip:
+          'You can check if your wallet extension is up to date. If update button is shown, please update it.',
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply src/i18n/en-US/index.ts]: @Kahonnohak Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/719/commits/f9780e1d3d6419d325b6e9de1bebfba7618d2972) commit.

**Review by Kahonnohak — APPROVED** [review]
LGTM!

**Pull request merged**